playbooks/code-review.md

```diff
@@ -2,3 +2,45 @@
 title: Code reviews
 description: How our code review process works 
 ---
+
+Reviewing other people's code is a responsibility of every single developer. Instead of viewing it as a hassle, it should be looked upon as a necessary process in order to keep the code clean and consistent.
+It is also an opportunity to learn something new.
+
+Rules on..
+
+## How a Code Review process should look like
+
+..are written below.
+
+### Terms:
+ - PR - Pull Request.
+ - PR Creator - a person who initiated the PR.
+ - Reviewer - a person assigned to do a PR code review.
+
+### Rules:
+1. Every PR has to be reviewed, with the exception of fix/hotfix PR-s that can be merged by the code owner on his own responsibility.
+2. When creating a PR, you should select **EXACTLY ONE** person as a reviewer. This person has to be someone who is working at the moment (not on vacation). 
+If there is someone else who should see this PR, he/she should be assigned as Assignee on this PR.
+	Reviewer Priority list:
+	1. Most senior developer on the project (excluding you).
+	2. Anyone else on the project.
+	3. Anyone else. (predetermined as a backup)
```

Comment:
I think "predetermined" is where it goes into vagueness and confusion, if we say "3) ask on this and this slack channel who is available" it is much more straightforward

Reply:
I somewhat agree with Tony.

What if the aforementioned backup person isn't able to perform code review? Does that mean that PR should be merged without code review? If so, it should be noted here. But I think that we can come up with a solution (i.e. random.org the names of all developers that are eligible for code review).
@markopaden what do you think?

Sidenote: I'm aware that there is a small possibility that not a single available person in our department can be found. And we should also note that possible outcome here.

Reply:
To continue from Brcko, this indeed needs more details. If we consider the types of developers in our agency... What happens in a such case where there is no available "PHP" developers, you surely shouldn't set a "JS" dev to review your PR then. @markopaden what do you think, can this be more precise?

Reply:
```suggestion
	3. Anyone else, find using slack channel [#development-codereview (https://bornfight.slack.com/archives/C013098CQ92) 🔒
```